master/docs/manual/cfg-www.rst

```diff
@@ -119,10 +119,10 @@ The available classes are described here:
 
     ::
 
-        from buildbot.plugins import util
+        from buildbot.www import auth
         c['www'] = {
             # ...
-            'auth': util.UserPasswordAuth({"homer": "doh!"}),
+            'auth': auth.UserPasswordAuth({"homer": "doh!"}),
         }
 
 .. py:class:: buildbot.www.auth.HTPasswdAuth(passwdFile)
```

Comment:
could be.
@sa2ajj  as you introduced plugin namespace you might have an opinion about it?